contracts/src/v0.7/tests/UpkeepPerformCounterRestrictive.sol

```diff
@@ -7,18 +7,29 @@ contract UpkeepPerformCounterRestrictive {
   uint256 public nextEligible = 0;
   uint256 public testRange;
   uint256 public averageEligibilityCadence;
-  uint256 count = 0;
+  uint256 public checkGasToBurn;
+  uint256 public performGasToBurn;
+  mapping(bytes32 => bool) public dummyMap; // used to force storage lookup
```

Comment:
private?

Reply:
it's just for tests, so I usually lean "leave it all public" for debugging purposes